doc/how-to/recover.md

```diff
@@ -0,0 +1,63 @@
+# MicroCloud Cluster Recovery
+
+Each MicroCloud service uses the [Dqlite](https://dqlite.io/) distributed
+database for highly-available storage. While the cluster recovery process is
+similar for each service, this document only covers cluster recovery for the
+`microcloudd` daemon. For cluster recovery procedures for LXD, MicroCeph and
```

Comment:
Just to confirm - is two 'd' correct?

Reply:
[Yeah](https://github.com/canonical/microcloud-pkg-snap/blob/latest-edge/snapcraft/commands/daemon.start#L4), my intention was to refer to just the microcloud daemon (to avoid ambiguity with the MicroCloud project/product).

Reply:
From an external user's point of view, is `microcloudd` ever actually exposed? Would they know what this is referring to?

Reply:
`snap services` (`microcloud.daemon`), or `ps -aux`; IMO an operator should be familiar with what is in scope/responsibilities for each tool (especially since almost all use of a deployed MicroCloud is done via LXD/Micro{Ceph,OVN}). I think `d` naming convention + context here makes the meaning clear.
You're welcome to disagree :)